modules/hostap/Kconfig

```diff
@@ -24,11 +24,8 @@ config WIFI_NM_WPA_SUPPLICANT
 if WIFI_NM_WPA_SUPPLICANT
 
 config HEAP_MEM_POOL_ADD_SIZE_HOSTAP
-	def_int 85000 if WIFI_NM_WPA_SUPPLICANT_CRYPTO_ENTERPRISE && !MBEDTLS_ENABLE_HEAP
-	def_int 85000 if WIFI_NM_HOSTAPD_AP
-	def_int 40000 if WIFI_NM_WPA_SUPPLICANT_AP
-	# 8192 for MbedTLS heap
-	def_int 21808 if MBEDTLS_ENABLE_HEAP
```

Comment:
why is this removed?

Reply:
Hi @krish2718  please see this commit, I think it's not correct:
https://github.com/zephyrproject-rtos/zephyr/commit/3ade4bed2749d2096bba293b8869c057adaf55a6
Originally hostap used libc heap:
(1) if MBEDTLS_ENABLE_HEAP is not defined, then hostap and mbedtls both use libc heap. 
(2) if MBEDTLS_ENABLE_HEAP is defined, then hostap uses libc heap, mbedtls uses its own heap.
therefore, this logic was correct for hostap using libc heap case:
![image](https://github.com/user-attachments/assets/eed02506-8f09-4d5e-bdc9-f9b0a8ee5118)
Since hostap starts to use kernel heap, but mbedtls don't use kernel heap, then we should not consider MBEDTLS_ENABLE_HEAP here.